### DISCUSSION TITLE
Is it true, that type: 'list' doesn't works in inline editing anymore?

### DISCUSSION THREAD

**Discussion by AlexVDem** [vitmalina/w2ui#2326] [Q&A] [answered]
In 2.0
"ERROR: input types "list", "enum" and "file" are not supported in inline editing."
What I have to use now instead "List" type? Combo do not fit, because can't returns selected item id. Or, can you give me the way, please, to get id of selected item in combo list?
Are you sure than 2.0 is upgrade of 1.5 but not downgrade? :)

**Comment by awadhesh4772** [comment]
I have same issue, field property as below.

{ field: 'ageunit', text: 'Age Unit', size: '75px', editable: { type: 'list', items: [{ id: 'Day', text: 'Day' },{ id: 'Month', text: 'Month' }] } },

It returns 'undefined' 

It was working on 1.5, and just upgraded to 2.0 and caught in this exception. :)

**Comment by vitmalina** [comment, marked as answer]
Sorry for a be-lated response. Here is a demo that shows `list` type working. https://w2ui.com/web/demos/#/grid/21. Enum is not working, though
